src/pages/Inventory.page.js

```diff
@@ -66,4 +72,31 @@ export class InventoryPage extends BaseSwagLabPage {
         let allNamesSorted = allNames.sort((a,b)=>b - a);
         expect(allNamesSorted).toEqual(allNames);
     }
+
+    async addRandomItemsToCart() {
+        let allNames = await this.page.locator('[data-test="inventory-item-name"]').allInnerTexts(); 
+        let selectedNames = [];
+        let allItesTogether= [];
+        for (let i = 0; i < 4; i++) {
+            const popedElement = this.popRandomElementFrom(allNames, selectedNames);
+            const price = await this.page.locator('[data-test="inventory-item-description"]').filter({ hasText: popedElement }).
```

Comment:
Let's create a function which will get the data (name, description and price) from the item with specific name? it will make your code easier to read.

Additionally, you can create private properties in class with selectors. E.g. `inventoryItemDescriptionSelector`, `inventoryItemNameSelector`, and one more and use it in the new function